### ISSUE TITLE
Update the Flatpak Install.md ref

### ISSUE THREAD

**Issue by sanecito** [mirukana/mirage#14]
Issue: When using the current instructions to install via Flatpak, `flatpak install --user flathub org.kde.Platform//5.12 mirage-*.flatpak`

Get the following:

```
Looking for matches…
error: No remote refs found similar to ‘flathub’
```

Solution: Update the cited reference to point to correct source

Other: Found this error using Manjaro OS

[Sounds like this will be resolved sometime next week](https://github.com/mirukana/mirage/pull/13#issuecomment-604721473).

**Comment by mirukana** [comment]
Does running `flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo` before the install command solves the issue?

**Comment by sanecito** [comment]
Ran that line, and then the one in Install.md again but still get:
```
Looking for matches…
error: No remote refs found similar to ‘flathub’
```
Let me disable my router VPN real quick and see if that somehow has any affect in finding ref

**Comment by mirukana** [comment]
Try these:
```
flatpak remote-add --user --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
flatpak install --user flathub org.kde.Platform//5.12
flatpak install --user mirage-*.flatpak
```

**Comment by sanecito** [comment]
Ah, this is probably a user error on my end. 

Not being super familiar with the cmdline flags I assumed it was hosted on flathub given flathub was mentioned. First command worked in that first didn't throw an error (but didn't ack it was successful either), second command downloaded a few missing kB/MB of several things, and then when I ran the third command I got:

`
error: Failed to open file “/home/<user>/mirage-*.flatpak”: open() failed: No such file or directory`

At which point I downloaded the 0.4.2 flatpak release from https://github.com/mirukana/mirage/releases and then copied it into my home directory. Reran the third command at which point it installed and launched w/o issue.
